lib/msprime.c

```diff
@@ -5870,9 +5854,6 @@ msp_dirac_common_ancestor_event(msp_t *self, population_id_t pop_id, label_id_t
         /* If no coalescence has occured, we need to signal this back to the calling
```

Comment:
This comment is also out of date, we should remove it now.

Reply:
@TPPSellinger, can you delete this comment that's referring to the old rejection code please? It'll be very confusing for people otherwise.

Reply:
My bad I forgot